pragma/core/mixins/randomness.py

```diff
@@ -256,3 +394,24 @@ async def handle_random(self, private_key: int, min_block: int = 0):
 
                 # Wait for Tx to pass
                 await asyncio.sleep(5)
+
+    async def get_last_example_random(self):
+        (response,) = await self.example_randomness.functions["get_last_random"].call()
+        return response
+
+    async def example_request_random(
```

Comment:
let's not add example contract related functions in the SDK. It's an external contract.